Dockerfile

```diff
@@ -1,16 +1,17 @@
 # Dockerfile
-FROM node:lts
+FROM node:16.13.1-alpine3.14
 
-WORKDIR /kodadot-app
+WORKDIR /app
 
-EXPOSE 9090
-
-COPY package.json /kodadot-app/package.json
+COPY package.json .
+COPY yarn.lock .
 
+RUN apk add --no-cache git
```

Comment:
Why do we need git package?

Reply:
Yarn needs `git` to function. It supports running `git` commands.
For more info: https://github.com/yarnpkg/yarn/issues/6881

Reply:
So how it was working without?

Reply:
What was working without?

Reply:
the docker compose

Reply:
Ah, I think you already knew about it since you are reviewing this PR :)

It's simple if you look at the old `docker-compose.yml`, it was using the local `node_modules`, which were installed using your local git.

```yml
version: '3.7'

services:
  frontend:
    container_name: Kodadot
    build:
      context: .
      dockerfile: Dockerfile
    volumes:
      - '.:/kodadot-app'
      - '/kodadot-app/node_modules'
    ports:
      - '0.0.0.0:9090:9090'
    environment:
      - CHOKIDAR_USEPOLLING=true
```